src/configs/Config.ts

```diff
@@ -76,5 +76,11 @@ export const config = {
       "primary_key": "id",
       "references": []
     }
-  }
-}
\ No newline at end of file
+  },
+  "exhaust_provider": process.env.exhaust_provider || "aws", // Supported providers - AWS, GCP, Azure
```

Comment:
Instead of exhaust_provider can we rename it as cloud_storage_provider? or csp_provider?